openwisp_monitoring/monitoring/base/models.py

```diff
@@ -544,6 +544,33 @@ def _default_query(self):
             q += default_chart_query[1]
         return q
 
+    @classmethod
+    def _get_group_map(cls, time=None):
+        """
+        Returns the chart group map for the specified days,
+        otherwise the default Chart.GROUP_MAP is returned
+        """
+        if not time or not isinstance(time, str) or time[-1] != 'd':
+            return cls.GROUP_MAP
+        group = '10m'
+        days = int(time.split('d')[0])
+        # Use copy of class variable to avoid unpredictable results
+        CUSTOM_GROUP_MAP = cls.GROUP_MAP.copy()
```

Comment:
I suggest turning this into lowercase: `custom_group_map`, because uppercase names usually indicate constants or values that do not change over the lifetime of the application.

Reply:
right :+1: